src/Bridge/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaRegexRestriction.php

```diff
@@ -29,7 +29,7 @@ class PropertySchemaRegexRestriction implements PropertySchemaRestrictionMetadat
      */
     public function create(Constraint $constraint, PropertyMetadata $propertyMetadata): array
     {
-        return isset($constraint->pattern) ? ['pattern' => $constraint->pattern] : [];
+        return $constraint->getHtmlPattern() ? ['pattern' => $constraint->getHtmlPattern()] : [];
```

Comment:
this is valid only `if ($constraint instanceof Regex)`

Reply:
Sure, but that is the \ApiPlatform\Core\Bridge\Symfony\Validator\Metadata\Property\Restriction\PropertySchemaRegexRestriction::supports method for, isn't?
```
public function supports(Constraint $constraint, PropertyMetadata $propertyMetadata): bool
    {
        return $constraint instanceof Regex && $constraint->match;
    }
```

Reply:
Anyway, instanceof check doesn't hurt, implemented.